routers/web/user/profile.go

```diff
@@ -70,12 +71,31 @@ func Profile(ctx *context.Context) {
 		uname = strings.TrimSuffix(uname, ".gpg")
 	}
 
+	isShowRSS := ""
+	if strings.HasSuffix(uname, ".rss") {
```

Comment:
Besides checking the suffix need adds checks of optional request media types: `application/atom+xml`, `application/rss+xml`